CODEOWNERS

```diff
@@ -0,0 +1 @@
+@dgaponov
```

Comment:
I think it should looks like `* dgaponov`

Reply:
Fixed